components/blocks/button.js

```diff
@@ -2,12 +2,20 @@ import Link from "next/link";
 
 import styles from "./button.module.css";
 
-const Button = ({ children, link }) => {
-  return (
-    <Link href={link}>
-      <button className={styles.Button}>{children}</button>
-    </Link>
-  );
+const Button = ({ children, link, target }) => {
+  if (target) {
```

Comment:
I wonder if we should explicitly check here if `target === '_blank`, since there are [other target options](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) besides `_blank` and `_self`.

I've hardly ever used them though, and they all behave like `_self`, so maybe it's not worth it?

Reply:
Oh good idea, that removes any ambiguity about what to set target to. Fixed on  ce80f8b.